### PR TITLE
Oz/s1283 readme custom actions

### PR DESCRIPTION
**Pull request by ozhey**

### CODE REVIEW

File: generate/generate.go [hunk 1]
```diff
@@ -232,20 +241,62 @@ func GenerateMarkdown(c *cli.Context) error {
...
 
+	if c.String("custom-actions") != "" {
+		generateCustomActionsReadme(f, c.String("custom-actions"))
+	}
+
 	return nil
 }
 
+func generateCustomActionsReadme(file *os.File, path string) {
+	type customAction struct {
```
Comment (YakirOren) [inline generate/generate.go:264]: use the struct from the blink-sdk

File: generate/generate.go [hunk 2]
```diff
@@ -232,20 +241,62 @@ func GenerateMarkdown(c *cli.Context) error {
...
+			Description string `yaml:"description"`
+		} `yaml:"parameters"`
+	}
+	currentDirectory, err := os.Getwd()
+	if err != nil {
+		log.Error(err.Error())
+		return
+	}
+	err = filepath.WalkDir(currentDirectory+path, func(filePath string, _ fs.DirEntry, err error) error {
+		if err != nil || !strings.HasSuffix(filePath, ".action.yaml") {
```
Comment (YakirOren) [inline generate/generate.go:277]: make const

Comment (ozhey) [reply generate/generate.go]: done

File: generate/generate.go (hunk 1 above)
Comment (ozhey) [reply generate/generate.go:265]: done

**Review by YakirOren — APPROVED** [review]

**Pull request merged**